src/views/TodoList/TodoList.js

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import { useState, useEffect } from 'react';
+import TaskForm from '../../components/TaskForm/TaskForm.js';
+import {
+  changeIsComplete,
+  createTaskItem,
+  deleteTask,
+  fetchTaskList,
+} from '../../services/fetchData.js';
+import './TodoList.css';
+
+export default function TodoList() {
+  const [task, setTask] = useState();
+  const [taskList, settaskList] = useState();
+  const [loading, setLoading] = useState(true);
+
+  useEffect(() => {
+    const fetchData = async () => {
+      const data = await fetchTaskList();
+      settaskList(data);
+      setLoading(false);
+    };
+    fetchData();
+  }, [loading]);
+
+  const handleSubmit = async (e) => {
+    e.preventDefault();
+    await createTaskItem(task);
+    setLoading(true);
```

Comment:
Really clever using loading to re-fetch the data. I might suggest keeping a consistent approach across the 3 todo handler functions though.